langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiStreamingChatModelIT.java

```diff
@@ -185,4 +196,160 @@ public void onError(Throwable error) {
         // then
         assertThat(response.content().text()).contains("Berlin");
     }
+
+    @Test
+    void should_accept_text_and_image_from_public_url() {
+
+        // given
+        Image image = Image.builder()
+                .url(CAT_IMAGE_URL)
+                .mimeType("image/png")
+                .build();
+        UserMessage userMessage = UserMessage.from(
+                TextContent.from("What do you see? Reply in one word."),
+                ImageContent.from(image)
+        );
+
+        // when
+        TestStreamingResponseHandler<AiMessage> handler = new TestStreamingResponseHandler<>();
+        visionModel.generate(singletonList(userMessage), handler);
+        Response<AiMessage> response = handler.get();
+
+        // then
+        assertThat(response.content().text()).containsIgnoringCase("cat");
+
+        assertThat(response.tokenUsage().inputTokenCount()).isEqualTo(268);
```

Comment:
I'm wondering if there's an update to the model, if the input token count could change? (thus making the assertion break)
I know images, regardless of their size, counts for a fixed amount of token. So the variability may come more from the short text itself.
Hopefully it'll just be a matter of updating the number, should it ever change some day.

Reply:
You're right, I will remove token assertions from here.